<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Part Inventory Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="previous_btn">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Inventory Details</string>
       </attribute>
       <widget class="QTableWidget" name="table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>951</width>
          <height>381</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;</string>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reference</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Part Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Min Area</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Max Area</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Num of Holes</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Min Diameter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Max Diamter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Count</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>651</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Search of references with count level lower or equal to:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="count_filter_txt">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="search_btn">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>0</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Inventory Statistics</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Refrence</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of parts type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>90</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Max. Number of Holes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Min. Number of Holes</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_ref_nbr">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_parts_nbr">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_nim_hole">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_max_hole">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>90</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_nim_hole_2">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Ref</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_max_hole_2">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>80</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Arial&quot;;
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Ref</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>761</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <column>
         <property name="text">
          <string>Reference</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Part Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Count</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>571</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Top 3 Reference with minimum inventory level</string>
        </property>
       </widget>
       <widget class="QPushButton" name="check_btn">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>190</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Edit Inventory</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Reference</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Part Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Area</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Area</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>210</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Diameter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>70</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Holes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>140</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Diameter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>270</y>
          <width>331</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="reference">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>101</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="part_name">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="min_area">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="max_area">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>300</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="number_of_holes">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>90</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="max_diameter">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>229</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="min_diameter">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>159</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="count">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>290</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="update_btn">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>340</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_btn">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>340</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_btn">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>340</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLabel" name="show_label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>15</y>
          <width>631</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(111, 111, 111);
color: rgb(0, 0, 255);
font: 75 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="update_txt">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(112, 112, 112);</string>
        </property>
        <property name="text">
         <string>Enter Item ID to Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ok_btn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="next_btn">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="refresh_btn">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 30pt &quot;Arial&quot;;
background-color: rgb(115, 115, 115);
color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Parts Inventory Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
